__tests__/__snapshots__/main.test.js.snap

```diff
@@ -82,6 +82,109 @@ Liran Tal
 
 
 
+<p align=\\"center\\"> rendered in postcontent </p>
+
+Thank you!
+
+
+<i>Updated on 2021-12-14T00:00:00.000Z</i>"
+`;
+
+exports[`given a directory of markdown files a full document is rendered with any yaml metadata passed 1`] = `
+"---
+title: a title
+description: decsription of document
+randomProp: a random prop description
+randomPropNumber: 5
+---
```

Comment:
This front matter part added to the README.md generated file is a requirement for those static site generators to create an HTML version of it then?

Reply:
No it's not needed after all (🥳) which is why I closed this PR.

And to answer you previous question, it was being displayed in the generated README file, which was another reason why it wasn't suitable.

But I found a way to skip this part completely with `11ty` and it's definitely not needed for jekyll.

I'm just in the process of updating the documenting it.

Reply:
Ok cool, I'll wait till you're finish with the changes you want to propose and we can follow up on that with more conversation :)